### ISSUE TITLE
reply to sharer email

### ISSUE THREAD

**Issue by mrozati** [Adobe-Marketing-Cloud/asset-share-commons#366]
**Use case**
Shared assets are sent usually from a no-reply email address. The receiver of email has currently no chance to contact back the sharer by email, if they don't know the sharer address.

**Enhancement: reply to sharer email**
If the sharer has an email address in his profile, it should be used in the Reply-To field of the email.
The editor should be able to turn this feature on and off
If the feature is turned off of if the sharer has no email, the field Reply-To remains empty. Most of email clients use the sender email as reply to, which is the default case.

**Comment by godanny86** [comment]
ah just saw this after the release:

```
 <reply-to-sharer-email
                                            jcr:primaryType="nt:unstructured"
                                            sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
                                            fieldDescription="Except for Anonymous, Admin or user's without Email, in which case no Reply-To address is used."
                                            text="Use Sharer's Email as Reply-To address"
                                            value="{Boolaen}false"
                                            name="./replyToSharer"/>
```

Bool**a**en vs. Bool**e**an

**Comment by godanny86** [comment]
will need to update the dialog and cut a new release

**Comment by godanny86** [comment]
closing this. The fix is tracked with #388
